apps/webClient/src/components/NavButton/index.tsx

```diff
@@ -1,43 +1,65 @@
 import React from "react";
+import clsx from "clsx";
 import { makeStyles } from "@material-ui/styles";
 import Button from "@material-ui/core/Button";
-import Typography from "@material-ui/core/Typography";
 import { Theme } from "@material-ui/core/styles/createMuiTheme";
 
 interface Props {
   route: string;
   text: string;
+  vertical?: boolean;
```

Comment:
extreme nitpick, no need to change, but as a general rule an `is` or `are` prefix is preferred on boolean

Reply:
Easy change. done